src/bot.py

```
@@ -5,785 +5,746 @@
-        await interaction.response.send_message(
-            f"[Invalid value for `{key}`. Expected type: `{target_type.__name__}`.]",
-            ephemeral=True,
+    @property
+    def config(self) -> RootConfig:
+        return config_manager.config
+
+    # I'm too stupid to find a different solution to strict type checking in local scopes
+    @property
+    def safe_user(self) -> discord.ClientUser:
+        if self.user is None:
+            raise RuntimeError("Bot user not initialized!")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)